src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './Introduce.scss';
+import { useNavigate } from 'react-router-dom';
+import { Link } from 'react-router-dom';
```

Comment:
```suggestion
import React from 'react';
import { Link, useNavigate } from 'react-router-dom';
import './Introduce.scss';
```

import 순서 수정해주세요! 일반적인 convention을 따라 순서만 잘 지켜주셔도 가독성이 좋아집니다. 
아래 순서 참고해주세요.

React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)